### PR TITLE
Migration/research v2

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**
This includes renaming the SageResearch v2 frameworks to ResearchV2 and ResearchV2UI. Additionally, I added in the MTB assessments using a similar "shoehorn" approach that was used for adding CRF. In other words, it is hard coded rather than using schedules for the AppConfig. This does not include the desired "show all 8 assessments" requirement b/c the requirements for doing so are unclear.

### CODE REVIEW

File: mPower2/mPower2/ActionBannerView.swift [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+// SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+// CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+// OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+// OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+
+import UIKit
+import ResearchV2UI
+
+// Use this just so the corner radius show's up in Interface Builder
```
Comment (Erin-Mounts) [inline mPower2/mPower2/ActionBannerView.swift:37]:
```suggestion
// Use this just so the corner radius shows up in Interface Builder
```

File: mPower2/mPower2/ActionBannerView.swift [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+@IBDesignable
+class ActionBannerView: UIView {
+//    @IBInspectable var cornerRadius: CGFloat = 0.0 {
```
Comment (Erin-Mounts) [inline mPower2/mPower2/ActionBannerView.swift:40]: Remove commented-out code

File: mPower2/mPower2/View Controllers/TaskBrowserViewController.swift [hunk 3]
```diff
@@ -522,11 +566,11 @@ class TaskCollectionViewCell: UICollectionViewCell {
...
 // Use this just so the corner radius show's up in Interface Builder
-@IBDesignable
-class RoundedCornerView: UIView {
-    @IBInspectable var cornerRadius: CGFloat = 0.0 {
-        didSet {
-            layer.cornerRadius = cornerRadius
-        }
-    }
-}
+//@IBDesignable
```
Comment (Erin-Mounts) [inline mPower2/mPower2/View Controllers/TaskBrowserViewController.swift:569]: TODO or delete commented out code

File: mPower2/mPower2TestApp/Info.plist [hunk 4]
```diff
@@ -15,9 +15,9 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>$(MARKETING_VERSION)</string>
+	<string>2.4</string>
```
Comment (Erin-Mounts) [inline mPower2/mPower2TestApp/Info.plist:18]: Was this (changing to hardcoded values) intentional?

**Review by Erin-Mounts — APPROVED** [review]
Couple of nits and/or questions, otherwise looks good

File: mPower2/mPower2TestApp/Info.plist (hunk 4 above)
Comment (syoung-smallwisdom) [reply mPower2/mPower2TestApp/Info.plist:18]: Yes, used the agvtool to set the marketing version

**Pull request merged**